time_tracker_api/projects/projects_namespace.py

```diff
@@ -19,15 +20,23 @@
     ),
     'description': fields.String(
         title='Description',
+        max_length=250,
         description='Description about the project',
         example=faker.paragraph(),
     ),
     'type': fields.String(
-        required=True,
+        required=False,
         title='Type',
-        max_length=30,
+        max_length=10,
         description='If it is `Costumer`, `Training` or other type',
-        example=faker.word(['Customer', 'Training']),
+        enum=PROJECT_TYPE.valid_type_values(),
+        example=faker.word(PROJECT_TYPE.valid_type_values()),
+    ),
+    'active': fields.Boolean(
```

Comment:
We can have a more flexible structure having a `status` field. This field can have values like:
- ACTIVE
- INACTIVE
- PAUSED, etc..

Reply:
You can put this in another ticket.